docs/docs/guide/google-tag-manager-and-growthbook.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: GrowthBook and Google Tag Manager
+description: This guide walks you through using GrowthBook with Google Tag Manager (GTM) to track your experiments and measure their impact on your business.
+sidebar_label: Google Tag Manager (GTM)
+slug: google-tag-manager-and-growthbook
+---
+
+# Using GrowthBook with Google Tag Manager (GTM)
+
+Now customers who are familiar with feature management using Google Tag Manager (GTM), yet may lack the engineering resources or capability to implement changes in their codebase, may easily use GrowthBook with GTM to power their AB tests. This setup is commonly used by marketing teams and CRO agencies.
+
+Note: GrowthBook also offers a [visual editor](https://docs.growthbook.io/app/visual) for lightweight, no-code UI changes.
+
+In this guide, we will assume familiarity with the GTM platform. We also assume the ability to apply a unique user ID via Google Analytics, cookies, etc; as well as the ability to track events (GA or otherwise).
+
+## Choosing a strategy: scalable and easy versus page speed
```

Comment:
I think in the majority of use cases, people will want the 2nd approach (inline experiments).  The wording in this guide makes it seem like it's really hard only and worth it if you need extreme performance. 

In reality, with GTM, inline experiments are not really any harder to implement (in fact it's often easier since you don't need to deal with API keys or configuring feature flag rules).  With both approaches, you need to deploy a new GTM version every time you create a new experiment.  The only difference is with feature flags, you can make changes later without another deploy.

I would at the very least, make the language less harsh in this section.  I would maybe even consider swapping the order and making Inline Experiments the default choice since I think it fits better with more people's use cases.